packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.scss

```diff
@@ -56,21 +52,7 @@
   }
 }
 
-.cards {
-  display: flex;
-  margin: 8px -20px;
-  flex-direction: column;
-
-  &.wide > * {
-    flex: 1 100%;
-  }
-}
-
-@media only screen and (max-width: 400px) {
-  .cards.wide > * {
-    flex: 1 320px;
-  }
-
+@media only screen and (max-width: 600px) {
```

Comment:
may be better to use our classes for screen-sizes:
screen-large, screen-medium...